api/src/main/java/com/github/eyefloaters/console/api/model/PartitionInfo.java

```diff
@@ -90,6 +98,18 @@ public Long getRecordCount() {
             .orElse(null);
     }
 
+    public Long getSize() {
```

Comment:
We should document the semantics of this. It could be the total size over all the replicas, or the size of one replica.

Reply:
I'll add a Javadoc comment. Here, it's only the log size from the leader.

Reply:
Maybe be even more explicit and call it `leaderLogSize` then?

Reply:
Renamed `getSize` on `PartitionInfo` to `leaderLocalStorage` and added a description to the API doc annotation about how it is derived.